molstar-extension/esbuild.config.js

```diff
@@ -0,0 +1,72 @@
+import { argv } from 'node:process';
+import * as esbuild from 'esbuild';
+import {sassPlugin} from 'esbuild-sass-plugin'
+
+const
+  productionMode = ('development' !== (argv[2] || process.env.NODE_ENV)),
+  target = 'chrome100,firefox100,safari15'.split(',');
+
+console.log(`${ productionMode ? 'production' : 'development' } build`);
+
+const buildCSS = await esbuild.context({
+  entryPoints: ['src/apps/viewer/main.css'],
+  bundle: true,
+  target,
+  // external: ['/images/*'],
+  loader: {
+    '.png': 'file',
+    '.jpg': 'file',
+    '.svg': 'dataurl'
+  },
+  logLevel: productionMode ? 'error' : 'info',
+  minify: true,
+  sourcemap: !productionMode && 'linked',
+  plugins: [sassPlugin()],
```

Comment:
Hmm, you actually do have it here. No need have `molstar.css` in site them.

Reply:
@dsehnal, you mean that I can use `main.css`
```
@import "../../node_modules/molstar/lib/mol-plugin-ui/skin/light.scss";
@import "../../node_modules/jsoneditor/dist/jsoneditor.css";
@import "../../node_modules/reactjs-popup/dist/index.css";
```


directly in `index.html` and do not need to compile them in `molstar.css`?

Reply:
Actaully, no. Should change this import `require('mol-plugin-ui/skin/light.scss');` .... I guess this should be `molstar/lib/...`, no?